lib/workers/bin/worker.rb

```diff
@@ -1,2 +1,10 @@
 type = ARGV.shift
+begin
+  # Remove the ::Runner class which is in all of them.
+  worker_name = type.gsub(/::Runner$/, '')
+  # Will set the process' name to eg. 'EVM Worker: MiqPriorityWorker'
+  # in ps aux, htop, ...
+  Process.setproctitle("EVM Worker: #{worker_name}")
```

Comment:
sorry - my code example was incomplete, we already do some stuff before the code I included. please keep that stuff